src/v/config/configuration.cc

```diff
@@ -161,6 +161,12 @@ configuration::configuration()
       required::no,
       64_MiB)
   , rack(*this, "rack", "Rack identifier", required::no, std::nullopt)
+  , dashboard_dir(
```

Comment:
Should this be a path, or a boolean? If it's a path, I think it implies that the user could move the dashboard project files around.
I thought this would be `enable_dashboard`, where the user can specify whether they wanna serve the dashboard or not, but the files would always be expected to exist at `/etc/redpanda/dashboard`.